.ci/azure-pipelines.yml

```
@@ -1,5 +1,7 @@
 variables:
   - group: 'jellyfin'
+  - ${{ if startsWith(variables['Build.SourceBranch'], 'refs/tags/v') }}:
```

```suggestion
  group: 'jellyfin'
  ${{ if startsWith(variables['Build.SourceBranch'], 'refs/tags/v') }}:
```

---

I don't think we can do that since "group" is an environment variable but telling Azure to use the vars defined in a group

---

hmmm, well my other guess, based on the log, would be this: 
```suggestion
  - group: 'jellyfin'
   ${{ if startsWith(variables['Build.SourceBranch'], 'refs/tags/v') }}:
     - JELLYFIN_VERSION: $[replace(variables['Build.SourceBranch'], 'refs/tags/', '')]
```

---

The if statement is basically a pre-processor thing, so it disappears and puts the indented block beneath it in it's place.

---

Seems to be working now but we can't now for sure until we publish the next release